free45/toplevels/bsg_ac_black_parrot_single_core_rc3/tcl/icc/dp_place_constraint_script.tcl

```diff
@@ -0,0 +1,73 @@
+puts "Flow-Info: Running script [info script]\n"
```

Comment:
Also tcl/icc is unused

Reply:
Same comment as before. However, should I just delete this folder? I am not sure if we would need these later?

Reply:
>dc_init, dc_spg and icc are all used only in the 40nm flow